airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/data/json/JsonToAirbyteValue.kt

```diff
@@ -40,10 +40,14 @@ class JsonToAirbyteValue {
                 is TimestampTypeWithTimezone,
                 is TimestampTypeWithoutTimezone -> TimestampValue(json.asText())
                 is UnionType -> toUnion(json, schema.options)
-                is UnknownType -> UnknownValue("From $schema: $json")
+                // If we fail to recognize the schema, just pass the json through directly.
+                // This enables us to more easily add new types, without breaking compatibility
+                // within existing connections.
+                is UnknownType -> fromJson(json)
```

Comment:
@johnny-schmidt you possibly have this fixed in one of your open PRs? I actually don't remember 😅

Reply:
I don't think so